common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -15,6 +15,36 @@ interface PackageJson {
   devDependencies: Record<string, string>;
 }
 
+/**
+ * Gets the proper configuration needed for rollup's commonJS plugin for @opentelemetry/api.
+ *
+ * NOTE: this manual configuration is only needed because OpenTelemetry uses an
+ * __exportStar downleveled helper function to declare its exports which confuses
```

Comment:
Interesting. That probably means that we'd always need something for opentelemetry in scenario like https://github.com/Azure/azure-sdk-for-js/issues/9899 when customers of SDK libraries try to rollup?

Reply:
Yes, although this might get better. 

@witemple-msft mentioned that the newer rollup commonjs plugin might do better in this situation, so hopefully the amount of configuration is reduced.

Reply:
(and I haven't tried this with other bundlers).